packages/e2e-tests/tests/export.spec.ts

```diff
@@ -45,8 +47,8 @@ test.describe("Export tests", () => {
         .locator('#app-root button:above(:text("Export"))')
         .first()
         .click()
-      await app.mainWin.locator(`text=${label}`).first().click()
```

Comment:
This was the one that needed revision. The text selector's behavior of searching for substrings (per [docs](https://playwright.dev/docs/selectors#text-selector)) was causing the test to click the "zjson" entry when it was trying to find the one for "json".

Reply:
much better